### ISSUE TITLE
how to on modifying the standard layout

### ISSUE THREAD

**Issue by georgmo** [vsco/keys#10]
hi

i like most of the standard layout, but would need some modifications as some things don't really work on my german-laid-out keyboard. i can workaround with autohotkey, but that doesn't work if shift or alt are pressed and its just a workaround.
i am also worried that i couldn't set up my own layout, as that looks pretty complicated.

so the easiest for me would just be to modify the existing (auto-installed) standard layout.
how would i do that?

thanks and thanks for the open source effort.
g

**Comment by adonnan** [comment]
Yes, I only want to make a few changes and additions. I am glad VSCO made this open source, but goodness they should/could have left us with more sample layouts to modify.

**Comment by nsfmc** [comment]
hi there, i've added the standard and simple keysets in my branch of this project https://github.com/nsfmc/keys/commit/a63e9bbb4fa35ac74777dfc83cc1b52a72ea618b 

hope that helps

**Comment by ablo** [comment]
Amazing!! I was just thinking of doing it myself because I could not find it anywhere.

Thanks!


On 20 maj 2016 at 16:22:39, Marcos Ojeda (notifications@github.com) wrote:

hi there, i've added the standard and simple keysets in my branch of this project nsfmc@a63e9bb

hope that helps
